app/router.js

```diff
@@ -8,6 +8,9 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:title' });
```

Comment:
So I've never had to put a variable with spaces into a route.  Will the router make this url friendly?  For example, if the title was "Beautiful House on the Beach" would the url end up being `/rentals/beautiful-house-on-the-beach` or would there be any code required to make this happen?   That's what I was getting at with the original suggestion to dasherize.